app/filter-helpers.js

```diff
@@ -10,19 +10,60 @@ for (var group in formatMappingData) {
 
 var sectionFilter = function (allContent, pageSection) {
   console.log("Filtering " + allContent.length + " items by " + pageSection);
+  var filteredList = _filteredList(allContent, pageSection);
```

Comment:
As discussed, these changes need to be applied to all pages, not just Childcare and Early Years. Not sure you need to extract this additional function.